abis/IFilteredMinterSEAV0.json

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
Should we update this repo to import the ABI npm package so that we don't need to manually import like this?

Reply:
This is a good callout. I would like to do this in a follow-on PR to isolate the change, and have added an issue for this here: https://github.com/ArtBlocks/artblocks-subgraph/issues/197

Reply:
SGTM!

Reply:
Thank you!! :)